client/src/document/on-github.tsx

```diff
@@ -62,7 +63,10 @@ MDN URL: https://developer.mozilla.org$PATHNAME
 * Folder: \`$FOLDER\`
 * MDN URL: https://developer.mozilla.org$PATHNAME
 * GitHub URL: $GITHUB_URL
+* Last commit hash: \`$HASH\`
```

Comment:
This is kinda hard to use. Most people will want to copy that URL and go to the commit logs in the GitHub to see what and what else happened in that commit. And if we display the commit permalink URL, you can still easily extract the SHA from the URL. 
```suggestion
* Last commit: https://github.com/$REPOSITORY/commit/$HASH
```

Reply:
Done! I also moved URL generation back to Node.